### DISCUSSION TITLE
Пайплайны

### DISCUSSION THREAD

**Discussion by KarimullinArthur** [hufrea/byedpi#124] [Ideas]
А сейчас у проекта релизы руками компайлятся?

Если шо, могу написать  пайплайн на гитхаб экшн или ещё где, и кинуть PR, заодно можно добавить сборки под фрибсд и что там ещё люди просили.

**Comment by denizzzka** [comment]
Хорошо бы ещё сделать скрипт-пробник разных (наиболее распространённых) вариантов

Только неясно как проверять работает или нет в автоматическом режиме

**Comment by KarimullinArthur** [comment]
Вообще хотя бы юнит тесты написать было бы неплохо.
Но и е2е в целом скостылить как-то можно, гнать поток с ютуба через проксю с российского йп, делать замер скорости чере проксю утилиты и без... ммм,  пахнет интересным костылём.

Но думаю даже как и сейчас есть, по тегу просто билдить и заливать в релайз бинари было бы здорово.

**Comment by hufrea** [comment]
> Если шо, могу написать пайплайн на гитхаб экшн или ещё где, и кинуть PR.

Было бы славно.

**Comment by alersrt** [comment]
И сразу сборочки под OPKG бахнуть, не забыв завести репы под это хозяйство.
